deploy/kind/audit.yaml

```diff
@@ -0,0 +1,15 @@
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  kubeadmConfigPatches:
+  - |
+    kind: ClusterConfiguration
+    metadata:
+      name: config
+    apiServer:
+      extraArgs:
+        "audit-dynamic-configuration": "true"
+        "feature-gates=DynamicAuditing": "true"
+        "runtime-config=auditregistration.k8s.io/v1alpha1": "true"
+- role: worker
```

Comment:
```suggestion
```
Is any particular reason why we need more than one node in this example? If no, I would remove this.

Reply:
Yes @leogr, because the provided example daemonset won’t schedule Falco daemon pods on the master nodes (due to the default tolerations set).
Otherwise we could add the master toleration. What do you think?

Reply:
I didn't consider that. You are right, both options are ok for me, thus I'm going to mark this as resolved.

Thanks!